HARK/ConsumptionSaving/tests/test_IndShockConsumerType.py

```diff
@@ -921,4 +921,4 @@ def test_calc_jacobian(self):
 
         self.assertAlmostEqual(CJAC_Perm.T[30][29], -0.06120, places=HARK_PRECISION)
         self.assertAlmostEqual(CJAC_Perm.T[30][30], 0.05307, places=HARK_PRECISION)
-        self.assertAlmostEqual(CJAC_Perm.T[30][31], 0.04674, places=HARK_PRECISION)
+        self.assertAlmostEqual(CJAC_Perm.T[30][31], 0.04674, places=HARK_PRECISION)
```

Comment:
Just one question. If this is fixing a bug in the jacobians, why do the tests not change?

Was it only the first row that was wrong? Did the bug not propagate to the other rows?

Reply:
First column was wrong.